src/utils/helpers.ts

```diff
@@ -34,7 +34,7 @@ export const tryParseJSON = (jsonString: string) => {
   try {
     return JSON.parse(jsonString)
   } catch (error) {
-    logger.error('Error parsing JSON: ', error as Error)
+    logger.error('Error parsing JSON')
```

Comment:
Why are we deleting the error here? Which will include the stack trace etc.

Reply:
Have added this comment now